fpm/src/fpm.F90

```diff
@@ -3,10 +3,31 @@ module fpm
 private
 public :: print_help, cmd_build
 
+integer, parameter :: os_linux = 1
+integer, parameter :: os_macos = 2
+integer, parameter :: os_windows = 3
+
 contains
 
+integer function get_os() result(r)
+#ifdef _WIN32
+    r = os_windows
+#elif defined(__APPLE__)
+    r = os_macos
+#elif defined(__linux__)
+    r = os_linux
+#else
+    ! Unsupported platform
+    r = -1
+    error stop
+#endif
+end function
+
 subroutine print_help()
+integer :: ostype
+ostype = get_os()
 print *, "Fortran Package Manager (fpm)"
+print *, ostype
```

Comment:
Unless you anticipate reusing `ostype` later, why not just `print *, get_os()`? (3 lines of code down to 1).

Reply:
I fixed that now.